test/Microsoft.AspNetCore.Cors.Test/CorsServiceTests.cs

```diff
@@ -881,6 +881,24 @@ public void ApplyResult_PreflightMaxAge_MaxAgeHeaderAdded()
             Assert.Equal("30", httpContext.Response.Headers["Access-Control-Max-Age"]);
         }
 
+        [Fact]
+        public void EvaluatePolicy_MultiOriginsPolicy_ReturnsVaryByOriginHeader()
```

Comment:
Update this test to have variations: one origin, multiple origins.
Add a negative test case where multiple origins and request from non-allowable origin doesn't have origin header